dev-python/pytest-describe/pytest-describe-0.11.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Describe-style plugin for py.test"
+HOMEPAGE="https://github.com/ropez/${PN} https://pypi.org/project/${PN}"
```

Comment:
My vim is painting this red — variables are forbidden inside HOMEPAGE.

Reply:
Can you please show me in the PMS where this is forbidden? I just had a quick view and for me, *SRC_URI* and *HOMEPAGE* are nearly defined the same: No one explicite states "variables are allowed" (or disallowed). Both variables are listed for cache file format so I would assume that both variables will be evaluated before cached and therefore variables in *HOMEPAGE* should work the same like variables in *SRC_URI*. But I am happy to learn something new.

Reply:
The package manager will accept variable references in HOMEPAGE, of course. Still, our policy is stricter: https://devmanual.gentoo.org/ebuild-writing/variables/index.html#ebuild-defined-variables

Reply:
Thanks, I'll update the ebuild(s).